numpy/lib/nanfunctions.py

```diff
@@ -877,7 +916,7 @@ def _nanmedian1d(arr1d, overwrite_input=False):
     See nanmedian for parameter usage
     """
     arr1d, overwrite_input = _remove_nan_1d(arr1d,
-        overwrite_input=overwrite_input)
+                                            overwrite_input=overwrite_input)
```

Comment:
nit: I preferred the previous indentation because it's more robust to variable renames.

Reply:
My IDE's auto-format did it. :/